cli/command/configure/cmd.go

```diff
@@ -120,13 +120,13 @@ func Run() error {
 	}
 
 	if !creds.IsValid() {
-		fmt.Println("Credentials provided looks invalid and won't be saved.")
-		return nil
+		log.Error().Msg("Credentials provided looks invalid and won't be saved.")
```

Comment:
Just a small suggestion to improve the existing message and make it a bit more assertive:
```suggestion
		log.Error().Msg("The provided credentials appear to be invalid and will NOT be saved.")
```